app/assets/stylesheets/custom.css.scss

```diff
@@ -158,3 +158,7 @@ table td {
   padding: 3px;
   margin-right: 10px;
 }
+
+.admin_orders_list h3 {
```

Comment:
Merge rule `.admin_orders_list h3` with rule on line 149<br>Selector `admin_orders_list` should be written in lowercase with hyphens